public/input_text_load_data.js

```diff
@@ -119,6 +146,43 @@ function loadData(collection) {
   });
 }
 
+// Changes XML to JSON
+xmlToJson = function(xml) {
+    var obj = {};
+    if (xml.nodeType == 1) {                
+        if (xml.attributes.length > 0) {
+            obj["@attributes"] = {};
+            for (var j = 0; j < xml.attributes.length; j++) {
+                var attribute = xml.attributes.item(j);
+                obj["@attributes"][attribute.nodeName] = attribute.nodeValue;
+            }
+        }
+    } else if (xml.nodeType == 3) { 
+        obj = xml.nodeValue;
+    }            
+    if (xml.hasChildNodes()) {
+        for (var i = 0; i < xml.childNodes.length; i++) {
+            var item = xml.childNodes.item(i);
+            var nodeName = item.nodeName;
+            if (typeof (obj[nodeName]) == "undefined") {
+                if(nodeName === 'sysStart' || nodeName === 'sysEnd' || nodeName === 'valStart' || nodeName === 'valEnd') {
```

Comment:
My document looked like this: 
     
![xmlexample](https://cloud.githubusercontent.com/assets/5742033/8942576/007b7d38-352b-11e5-8903-70e19ab5497e.png)

So the sysStart, valStart etc were within the object property "record", but I wanted them to be their own properties.